[2024-02-12T14:16:40.321+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_DAG_V4.17.data_to_db scheduled__2024-02-12T00:56:00+00:00 [queued]>
[2024-02-12T14:16:40.388+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_DAG_V4.17.data_to_db scheduled__2024-02-12T00:56:00+00:00 [queued]>
[2024-02-12T14:16:40.391+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-12T14:16:40.470+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): data_to_db> on 2024-02-12 00:56:00+00:00
[2024-02-12T14:16:40.486+0000] {standard_task_runner.py:60} INFO - Started process 20021 to run task
[2024-02-12T14:16:40.519+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Weather_DAG_V4.17', 'data_to_db', 'scheduled__2024-02-12T00:56:00+00:00', '--job-id', '10661', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmpwizddfce']
[2024-02-12T14:16:40.539+0000] {standard_task_runner.py:88} INFO - Job 10661: Subtask data_to_db
[2024-02-12T14:16:40.800+0000] {task_command.py:423} INFO - Running <TaskInstance: Weather_DAG_V4.17.data_to_db scheduled__2024-02-12T00:56:00+00:00 [running]> on host 06573eafe810
[2024-02-12T14:16:41.474+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_DAG_V4.17' AIRFLOW_CTX_TASK_ID='data_to_db' AIRFLOW_CTX_EXECUTION_DATE='2024-02-12T00:56:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-12T00:56:00+00:00'
[2024-02-12T14:16:41.503+0000] {postgres.py:172} INFO - Running copy expert: COPY weather_tab FROM STDIN, filename: weather.csv
[2024-02-12T14:16:41.614+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-12T14:16:41.653+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-12T14:16:41.689+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Weather_DAG_V4.17, task_id=data_to_db, execution_date=20240212T005600, start_date=20240212T141640, end_date=20240212T141641
[2024-02-12T14:16:41.917+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-12T14:16:42.110+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
